utils/src/main/scala/it/pagopa/interop/commons/logging/logging.scala

```diff
@@ -25,6 +34,7 @@ package object logging {
       MDC.put(SUB, contextOrBlank(fields, SUB))
       MDC.put(ORGANIZATION_ID_CLAIM, contextOrBlank(fields, ORGANIZATION_ID_CLAIM))
       MDC.put(CORRELATION_ID_HEADER, contextOrBlank(fields, CORRELATION_ID_HEADER))
+      MDC.put(ACCEPT_LANGUAGE, contextOrBlank(fields, ACCEPT_LANGUAGE))
```

Comment:
if I recall, this is used only to add specific information to every log, for example `[CID=abc] [OID=1234] A log message`
I don't think we need to log the requested language for each log, if needed we will add it to specific logs (eg when creating a document)